### PR TITLE
Use common types in frontend and backend

### PR DESCRIPTION
**Pull request by DadImScared**

### CODE REVIEW

**Comment by DadImScared** [comment]
[Use common types in frontend and backend](https://trello.com/c/KY6HRaiM/116-use-common-types-in-frontend-and-backend)

File: frontend/src/mixins/UsersByRole.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import fb from "@/firebaseApp";
-import { IUser } from "@/types/Users";
+import { IUserVueFire } from "@/types/Users";
 
 const filteredStatus = ["Lost", "Opted out", "Incorrect", "Duplicate"];
 
 @Component
 export default class UserByRole extends Vue {
-  users: IUser[] | null = null;
+  users: IUserVueFire[] | null = null;
```
[the remark this answers is not in the document]
Comment (DadImScared) [reply frontend/src/mixins/UsersByRole.ts]: Sometimes ".key" is used as that's the id of the item when VueFire converts the object of objects to an object of arrays with bindAsArray.

[the remark this answers is not in the document]
Comment (anantakrishna) [reply frontend/src/mixins/UsersByRole.ts]: That's okay, but can't it be dynamic? Should it be declared explicitly in the types?

[the remark this answers is not in the document]
Comment (DadImScared) [reply frontend/src/mixins/UsersByRole.ts]: Since it's used in the code I think it should.

**Pull request closed without merging**